src/react-components/src/components/02-molecules/Cards/EditionCard.tsx

```diff
@@ -28,21 +29,30 @@ export interface EditionCardProps {
 
   readOnlineLink?: string;
   downloadLink?: string;
+
+  noLinkElement?: JSX.Element;
 }
 
 /**
  * EditionCard component that renders information for an edition.
  */
 export default function EditionCard(props: React.PropsWithChildren<EditionCardProps>) {
-  const { id, blockName, modifiers = [], coverUrl,
+  const {
+    id,
+    blockName,
+    modifiers = [],
+    coverUrl,
     editionHeadingElement,
     editionInfo = [],
-    readOnlineLink, downloadLink } = props;
+    readOnlineLink,
+    downloadLink,
+    noLinkElement = <>Unavailable to read online</>
+  } = props;
   const baseClass = "edition-card";
 
   const getButtonsElement = (readOnlineLink: string, downloadLink: string, baseClass: string) => {
     if (!readOnlineLink && !downloadLink) {
-      return <div className={(bem("missing-links", [], baseClass))}>Unavailable to read online</div>;
+      return <div className={(bem("missing-links", [], baseClass))}>{noLinkElement}</div>;
```

Comment:
This is just me thinking outloud: I would expect `getButtonsElement` to return just a div of buttons or null. Might it make more sense to have something like `const btns = !readOnlineLink && !downloadLink ? getButtonElement(...) : noLinksDiv;` where `noLinksDiv` is `<div className={(bem("missing-links", [], baseClass))}>{noLinkElement}</div>;`. It's not a big deal and this implementation is fine.